src/net_processing.cpp

```diff
@@ -4636,7 +4632,6 @@ bool PeerManager::SendMessages(CNode* pto)
             !pto->HasPermission(PF_FORCERELAY) // peers with the forcerelay permission should not filter txs to us
         ) {
             CAmount currentFilter = m_mempool.GetMinFee(gArgs.GetArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE) * 1000000).GetFeePerK();
-            int64_t timeNow = GetTimeMicros();
```

Comment:
Is there any reason to subtract the time it took to send previous messages from the next send? It probably doesn't matter too much, because we assume that sending all messages is instantaneous, but getting the current time is also trivial complexity, so I have a slight preference to keep the getter here.

Reply:
I generally thing we're making way too many time calls already, and I doubt microsecond precisions matters at all for the fee filter sending.